backend/protocol_rpc/transactions_parser.py

```
@@ -291,6 +291,26 @@ def decode_deployment_data(self, data: str) -> DecodedDeploymentData:
+        Args:
+            raw_transaction: The hex-encoded signed transaction from Metamask
+            
+        Returns:
+            str: The transaction hash as hex string with 0x prefix
+        """
+        try:
+            transaction_bytes = HexBytes(raw_transaction)
+            signed_transaction = Transaction.from_bytes(transaction_bytes)
+            hash_bytes = self._hash_of_signed_transaction(signed_transaction)
+            return "0x" + hash_bytes.hex()
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)